statusHandler/termuiStatusHandler.go

```diff
@@ -0,0 +1,54 @@
+package statusHandler
+
+import "github.com/ElrondNetwork/elrond-go/termuic"
+
+// TermuiStatusHandler will be used when an AppStatusHandler is required, but another one isn't necessary or available
+type TermuiStatusHandler struct {
+	tui *termuic.TermuiConsole
+}
+
+// NewTermuiStatusHandler will return an instance of the struct
+func NewTermuiStatusHandler() *TermuiStatusHandler {
+
+	tsh := new(TermuiStatusHandler)
+	tsh.tui = termuic.NewTermuiConsole()
+	tsh.tui.Start()
+	return tsh
+}
+
+//Termui method - returns address of TermuiConsole structure from TermuiStatusHandler
+func (tsh *TermuiStatusHandler) Termui() (*termuic.TermuiConsole, error) {
+	if tsh.tui == nil {
```

Comment:
How can we have nil tsh.tui here?
Should consider removing this check and error return parameter

Reply:
Yea we can't have nil there. Removed useless verification and error.